parlai/tasks/natural_questions/agents.py

```diff
@@ -277,5 +305,5 @@ def __init__(self, opt, shared=None):
         super().__init__(opt, shared)
 
 
-class DefaultTeacher(NaturalQuestionsTeacherShortAnswer):
```

Comment:
I am not using this old version. And I not sure if anyone else does. I guess it should be OK to remove, or add a warning for deprecation in like 2 month if someone imports it.